docs/integrations/saas-cloud/tenable.md

```diff
@@ -0,0 +1,72 @@
+- **Risk Reduction**. Identify and mitigate high-severity vulnerabilities and security threats promptly, reducing the risk of security breaches.
+- **Efficient Remediation**. Locate affected hosts and prioritize remediation efforts, streamlining the process of securing your environment.
+- **Compliance Assurance**. Simplify compliance monitoring and reporting by tracking relevant audit events and security controls.
+- **Real-time Insights**. Gain real-time insights into your security environment, enabling rapid incident response and threat containment.
+
+## Log types
+
+This app uses [Tenable](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/tenable-source/) source to collect [Vulnerabilities data](https://developer.tenable.com/reference/exports-vulns-request-export) and [Audit Events](https://developer.tenable.com/reference/audit-log-events) from the Tenable platform.
+
+## Sample log message
+
+Refer to the Tenable API documentation for [Vulnerabilities data](https://developer.tenable.com/reference/exports-vulns-request-export) and [Audit Events](https://developer.tenable.com/reference/audit-log-events) log messages.
+
+## Sample Query
+
+```sql title="Vulnerability Events"
+(_source=Tenable fqdn)
+| json "asset.device_type", "asset.fqdn" ,"asset.hostname", "asset.ipv4", "asset.ipv6", "asset.last_authenticated_results", "asset.operating_system" , "output", "plugin.family" , "plugin.type" , "plugin.name", "plugin.risk_factor", "plugin.synopsis", "plugin.exploit_available" , "plugin.exploited_by_malware", "plugin.solution", "state", "scan.completed_at", "port.port", "port.protocol", "last_found" as device_type, fqdn, hostname, host_ipv4, host_ipv6, last_authenticated_scan_time, host_os, output,plugin_family, plugin_type ,plugin_name, plugin_risk_factor, plugin_synopsis, plugin_exploit_available, plugin_exploited_by_malware, plugin_solution, state, scan_completed_at, port, protocol, last_scan_time nodrop
+```
+
+```sql title="Audit Events"
+_sourceCategory=Tenable
+| json "description", "actor.name", "target.name", "action" as description, actor_name, target_name, action
+```
+
+## Set up collection
+
+To set up [Cloud to Cloud Integration Tenable Source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/tenable-source/) for the Tenable app, follow the instructions provided. These instructions will guide you through the process of creating a source using the Tenable source category, which you will need to use when installing the app. By following these steps, you can ensure that your Tenable app is properly integrated and configured to collect and analyze your Tenable data.
+
+## Installing the Tenable app​
+
+This section has instructions for installing the Symantec Web Security Service App for Sumo Logic.
+
+{@import ../../reuse/apps/app-install.md}
+
+## Viewing Tenable dashboards​
```

Comment:
The screenshots are broken for the dashboards "Tenable - Overview" and "Tenable - Audit Activity". When I accessed the image links directly, it gave me an "Access Denied" error. Also, the URLs for both dashboards appear to be the same. Could you make sure those image URLs are correct?

Reply:
Overview dashboard link should be https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Tenable/Tenable-Overview.png